epoxy-sample/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'android-apt'
 
 android {
   compileSdkVersion 24
-  buildToolsVersion "24.0.1"
+  buildToolsVersion "24.0.2"
```

Comment:
One of the less fun tasks I need to do is consolidate all dependency versions into one place, so when we need to update one of them we only do it once. I like the way butterknife does it, for example.

If you're interested that would be awesome. I need to make an issue about it